cogboard-webapp/src/actions/thunks.js

```diff
@@ -100,8 +100,13 @@ const makeWidgetUpdaterThunk = (beforeUpdateActionCreator, widgetDataCreator) =>
 const removeWidgetThunk = (id) =>
   (dispatch, getState) => {
     const { currentBoard: boardId } = getState().ui;
+    const token = getState().app.jwToken;
 
-    dispatch(deleteWidget(id, boardId));
+    return fetchData(URL.DELETE_WIDGET, 'POST', {id: id}, token)
```

Comment:
You don't need to repeat objects props with the same name. { id } will be enough.